manim/mobject/types/vectorized_mobject.py

```diff
@@ -1023,6 +1049,50 @@ def point_from_proportion(self, alpha: float) -> np.ndarray:
 
             current_length += length
 
+    def proportion_from_point(
+        self,
+        point: typing.Iterable[typing.Union[float, int]],
+    ) -> float:
```

Comment:
This should probably also be put in `Mobject` and raise a `NotImplementedError` like `point_from_proportion` currently is.

Reply:
Agreed; done.